### ISSUE TITLE
[A11yInspect] Fail: 2.1.1 A in /home/runner/work/test-app/test-app/src/App.tsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#26]
## ❌ Accessibility Fail

**Rule:** 2.1.1 A
**File:** `/home/runner/work/test-app/test-app/src/App.tsx`
**Line:** 51, **Column:** 1

### Description
Button fails to receive keyboard focus (Blocker)

### Context
```html
<div role="button" aria-checked="true">Fake toggle</div>
```

### Selector
`html > body > div > audio > div:nth-of-type(2)`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
